src/tasks/S3Upload/task.json

```
@@ -4,7 +4,7 @@
     "friendlyName": "Amazon S3 Upload",
     "description": "Upload file and folder content to an Amazon Simple Storage Service (S3) Bucket on AWS",
     "author": "Amazon Web Services",
-    "helpMarkDown": "Please refer to [Working with Amazon S3 Buckets](https://docs.aws.amazon.com/AmazonS3/latest/dev/UsingBucket.html) for more information on working with Amazon S3.\n\nMore information on this task can be found in the [task reference](https://docs.aws.amazon.com/vsts/latest/userguide/s3-upload.html).\n\n####Task Permissions\nThis task requires permissions to call the following AWS service APIs (depending on selected task options, not all APIs may be used):\n* s3:CreateBucket\n* s3:HeadBucket\n\nContent uploads are performed using S3's PutObject API and/or the multi-part upload APIs. The specific APIs used depend on the size of the individual files being uploaded.",
+    "helpMarkDown": "Please refer to [Working with Amazon S3 Buckets](https://docs.aws.amazon.com/AmazonS3/latest/dev/UsingBucket.html) for more information on working with Amazon S3.\n\nMore information on this task can be found in the [task reference](https://docs.aws.amazon.com/vsts/latest/userguide/s3-upload.html).\n\n####Task Permissions\nThis task requires permissions to call the following AWS service APIs (depending on selected task options, not all APIs may be used):\n* s3:CreateBucket\n* s3:ListBucket\n\nContent uploads are performed using S3's PutObject API and/or the multi-part upload APIs. The specific APIs used depend on the size of the individual files being uploaded.",
```

Thanks for your contribution! I believe the current wording is accurate as it is specifically referring to an API operation and not an IAM permission (s3:HeadBucket is an operation). Instead, maybe it is useful to link to the page to see the required permissions to perform this action.
```suggestion
    "helpMarkDown": "Please refer to [Working with Amazon S3 Buckets](https://docs.aws.amazon.com/AmazonS3/latest/dev/UsingBucket.html) for more information on working with Amazon S3.\n\nMore information on this task can be found in the [task reference](https://docs.aws.amazon.com/vsts/latest/userguide/s3-upload.html).\n\n####Task Permissions\nThis task requires permissions to call the following AWS service APIs (depending on selected task options, not all APIs may be used):\n* s3:CreateBucket\n* [s3:HeadBucket](https://docs.aws.amazon.com/AmazonS3/latest/API/API_HeadBucket.html)\n\nContent uploads are performed using S3's PutObject API and/or the multi-part upload APIs. The specific APIs used depend on the size of the individual files being uploaded.",
```